crime-commons-mod-schemas/src/main/resources/schemas/meansassessment/common/apiIncomeEvidence.json

```diff
@@ -9,6 +9,10 @@
       "type": "integer",
       "description": "Evidence id"
     },
+    "fias_id": {
+      "type": "integer",
+      "description": "Financial Assessment System (FIAS)id"
+    },
```

Comment:
Isn't this redundant if these records will be returned as a child of the financial assessment data structure?